.gitignore

```diff
@@ -127,8 +127,6 @@ packages/web/app/next.config.mjs
 packages/web/app/environment-*.mjs
 packages/web/app/src/gql/*.ts
 packages/web/app/src/gql/*.json
-# Changelog
-packages/web/app/src/components/ui/changelog/generated-changelog.ts
```

Comment:
I think it's better to save to git, to actually see what's happens with content of this file